examples/jsm/loaders/FBXLoader.js

```diff
@@ -2447,7 +2447,7 @@ class AnimationParser {
 
 					curveNodesMap.get( animationCurveID ).curves[ 'z' ] = animationCurve;
 
-				} else if ( animationCurveRelationship.match( /d|DeformPercent/ ) && curveNodesMap.has( animationCurveID ) ) {
+				} else if ( animationCurveRelationship.match( /d\|DeformPercent/ ) && curveNodesMap.has( animationCurveID ) ) {
```

Comment:
Original regex either matched `d` or `DeformPercent`. New regex matches only `d|DeformPercent`. Is this a correct fix?

Reply:
yes, console.log `animationCurveRelationship` varaible is `d|DeformPercent`

Reply:
![image](https://user-images.githubusercontent.com/40897360/226080787-2605b05f-d0ef-49d6-8072-97b1cce31391.png)

![image](https://user-images.githubusercontent.com/40897360/226080723-84ce0939-88bb-4823-a19b-5c0a337a6a94.png)

here is the variable print.
I have a custom attribute call `TimeCodeFrame` would match to morph curve.

Reply:
I know FBX has no official spec but do you know an inofficial resource that explains the `d|*` naming pattern?

Reply:
BTW: Would it also work to just use `match( /DeformPercent/ )`? 

Besides, do you mind sharing a FBX asset for testing?

Reply:
[fbx_morph.zip](https://github.com/mrdoob/three.js/files/11008842/fbx_morph.zip)

here is a simple issue FBX , it just a marker add a custom attribute call `TimeCodeFrame`.
I export it from maya.

![image](https://user-images.githubusercontent.com/40897360/226113607-84212102-6423-429a-b19d-57aae302df23.png)

FBXLoader would load it fail. I trace the issue and find out the `DeformPercent` issue.

I try to convert the fbx into ascii，and I found out the fbx properties add `d|` prefix

![image](https://user-images.githubusercontent.com/40897360/226113729-55659bb1-64bb-46d8-b681-43696e16d129.png)

Reply:
@Mugen87  
Is that ready to merge?

Reply:
I would prefer to use just `/DeformPercent/` as the regex. The other regular expressions in `parseAnimationCurves()` also discard the `d|` so I don't think we should honor it in just on check.

Reply:
Updated the PR accordingly.